deploy.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Deployer;
+
+require 'recipe/common.php';
+
+/*
+ * REDAXO Configuration
+ */
+
+$baseDir = dirname(dirname(dirname(dirname(__DIR__))));
+if (0 === strpos($baseDir, getcwd())) {
+    $baseDir = substr($baseDir, strlen(getcwd()));
+    $baseDir = ltrim($baseDir.'/', '/');
+}
+
+set('shared_dirs', [
+    $baseDir.'media',
+    $baseDir.'redaxo/data/addons/cronjob',
+    $baseDir.'redaxo/data/addons/phpmailer',
+    $baseDir.'redaxo/data/core',
```

Comment:
Man kann schon zurück, der vorherige Release-Ordner ist ja noch da, und auch dort zeigen die Symlinks ja auf die Ordner in `shared`.